tutorminio/patches/local-docker-compose-dev-services

```diff
@@ -1,6 +1,6 @@
 minio:
   ports:
-    - "127.0.0.1:9000:9000"
+    - "127.0.0.1:9001:9001"
```

Comment:
:+1: could you amend the README accordingly?

Reply:
Done

Reply:
See my comment above: we might have to expose both the 9000 and the 9001 ports.